src/test/scala/com/getvisibility/learning/lists/ListsTest.scala

```diff
@@ -28,4 +28,6 @@ class ListsTest extends AnyWordSpec {
       }
     }
   }
+
+  // TODO please check that all methods have unit tests
```

Comment:
Unwanted comment, so we can remove it.

Reply:
Will do. But I'm very fond of my comments, and you've broken my heart.